plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/PluginInteractionCallback.java

```diff
@@ -25,5 +25,5 @@
 
     Map<String, String> requestHeaders(String resolvedExtensionVersion);
 
-    T onSuccess(String responseBody, String resolvedExtensionVersion);
+    T onSuccess(String responseBody, Map<String, String> responseHeaders, String resolvedExtensionVersion);
```

Comment:
This signature changed touched a lot of files too, so you will see a lot of files that just add a parameter. Generally safe. The one place where this matters is in `JsonMessageHandler2_0` -- specifically in `responseMessageForPipelineExport()`, we use the responseHeaders to construct a `ExportedConfig` object.